### PR TITLE
Editorial and platform updates

### PR DESCRIPTION
**Pull request by SirSpidey**
- Change images to .svg
- Update links to docs framework
- Add search metadata

### CODE REVIEW

File: best-practices.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 copyright:
   years: 2015, 2019
-lastupdated: "2019-02-01"
+lastupdated: "2019-03-06"
+
+keywords: language support,supported languages,best practices,guidelines,
```
Comment (jeffpk62) [inline best-practices.md]: Remove trailing comma?

File: tool-overview.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
 
 copyright:
   years: 2015, 2019
-lastupdated: "2019-02-01"
+lastupdated: "2019-03-06"
+
+keywords: Watson Studio,Natural Language Classifier tool,gui,user interface
```
Comment (jeffpk62) [inline tool-overview.md]: I don't know that it matters, but should `gui` be in all-caps?  Same with `csv` in the next file.  Probably irrelevant but thought I'd ask.

Comment (SirSpidey) [reply tool-overview.md]: I have no idea and I'm not worrying about it. I figured I'd get something and then work through some similar questions I had with the Cloud team.

Comment (jeffpk62) [reply tool-overview.md]: That's the best answer yet!

**Pull request merged**